test/integration/scripting_spec.mjs

```diff
@@ -1535,49 +1535,35 @@ describe("Interaction", () => {
 
     it("must check that charLimit is correctly set", async () => {
       await Promise.all(
-        pages.map(async ([browserName, page]) => {
+        pages.map(async ([, page]) => {
           await page.waitForFunction(
             "window.PDFViewerApplication.scriptingReady === true"
           );
 
-          await clearInput(page, getSelector("7R"));
```

Comment:
This line is removed because the input field is already empty if the document is opened, so clearing it again is not necessary.